### ISSUE TITLE
State class total_increasing, but its state is not strictly increasing

### ISSUE THREAD

**Issue by seanmccabe** [garbled1/homeassistant_ecowitt#83]
Got the following warning in Home Assistant log:

_Entity sensor.rain_rate_2 from integration ecowitt has state class total_increasing, but its state is not strictly increasing. Please report it to the custom component author._

```
Logger: homeassistant.components.sensor.recorder
Source: components/sensor/recorder.py:309
Integration: Sensor (documentation, issues)
First occurred: November 21, 2021, 7:05:10 PM (1 occurrences)
Last logged: November 21, 2021, 7:05:10 PM

Entity sensor.rain_rate_2 from integration ecowitt has state class total_increasing, but its state is not strictly increasing. Please report it to the custom component author.
```

**Comment by garbled1** [comment]
huh.. thats.. odd.. I need to dig into this again.  in AZ I never get rain..

**Comment by seanmccabe** [comment]
No worries, in NZ and its Spring, lots of opportunity for rain.
If you do a potential fix, happy to test and report back.

**Comment by garbled1** [comment]
Can I ask, which rain rate is that?  Is that the current rate? Not any of the monthly/weekly/daily phizz?  If so, yeah, I think the fix is to set that one to measurement.

**Comment by seanmccabe** [comment]
Yer its the current rate, just a 2nd one as measured in mm here.

**Comment by seanmccabe** [comment]
As far as I can tell on the graph, it was still raining, but it looks like the rain rate decreased for a time.

**Comment by garbled1** [comment]
Yep, my bad, forgot that one should be measurement.  Will fix when I have a sec.  Thx!

**Comment by seanmccabe** [comment]
> Yep, my bad, forgot that one should be measurement. Will fix when I have a sec. Thx!

Any luck on getting this one fixed? The log entry is super annoying every time it rains.
